src/common/OptionButton.js

```diff
@@ -20,11 +20,15 @@ const styles = StyleSheet.create({
   rightIcon: {
     marginRight: 8,
   },
+  leftIcon: {
```

Comment:
Just check if we can reuse `rightIcon` style here. (by renaming it) 
(or even can we place it in `theme.js`)
(As it is also been added in `src/common/OptionRow.js`)

Reply:
@jainkuniya You are right. I'll add a single instance of the `rightIcon` in `theme.js`
Although, now that you mention it, I think the `optionRow` style can also be moved to `theme.js` since that too is being duplicated. Shall I move that too?